device/mellanox/x86_64-nvidia_sn5640-r0/Mellanox-SN5640-C512S2/buffers_defaults_t1.j2

```diff
@@ -0,0 +1,54 @@
+{#
+    SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
+    Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+    Apache-2.0
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+{% set default_cable = '0m' %}
+{%-set ports2cable = {
+            'torrouter_server'       : '0m',
+            'leafrouter_torrouter'   : '0m',
+            'spinerouter_leafrouter' : '0m'
+       }
+-%}
+{% set ingress_lossless_pool_size =  '105420800' %}
+{% set ingress_lossless_pool_xoff =  '0' %}
+{% set egress_lossless_pool_size =  '136210432' %}
```

Comment:
where does this number come from? I see the maximum pool is 709427 cells which is 136209984 bytes